server/src/sylvester/sdl/Query.graphql

```diff
@@ -0,0 +1,71 @@
+# ReNFT Escrow contract events are served to be consumed
+# Using these events, the on-chain state can be reconstructed anywhere
+
+enum EventType {
+  LEND
+  RENT
+  STOP_RENT
+  STOP_LEND
+  RENT_CLAIMED
+}
+
+interface Event {
+  tx_hash: String!
+  tx_offset: Int!
+}
+
+type Lend implements Event {
+  tx_hash: String!
+  tx_offset: Int!
+  is721: Boolean!
+  lenderAddress: String!
+  nftAddress: String!
+  tokenID: String!
+  lendingID: Int!
+  maxRentDuration: Int!
+  dailyRentPrice: Float!
+  lendAmount: Int!
+  paymentToken: Int!
+}
+
+type Rent implements Event {
+  tx_hash: String!
+  tx_offset: Int!
+  lendingID: Int!
+  rentingID: Int!
+  rentAmount: Int!
+  rentDuration: Int!
+  rentedAt: Int!
+}
+
+type StopRent implements Event {
+  tx_hash: String!
+  tx_offset: Int!
+  rentingID: Int!
+  stoppedAt: Int!
+}
+
+type StopLend implements Event {
+  tx_hash: String!
```

Comment:
some are snake case and some are camel case. Let's stick to one convention. Since we are in python, let's go with snake case.